locales/base/translation.json

```diff
@@ -1395,6 +1395,8 @@
       "header": "Success",
       "title": "Your funds are on their way!",
       "description": "Your transaction request has been sent & your funds will arrive in {{duration}}.",
+      "descriptionNumDays": "Your transaction request has been sent & your funds will arrive in 1-3 days.",
+      "description24Hours": "Your transaction request has been sent & your funds will arrive within 24 hours.",
```

Comment:
Wasn't sure if we can dynamically fill in duration for within vs in (especially with other language translations), so explicitly added these strings similar to what we have for selectProvidersScreen strings. Eventually when we do dynamic strings based on quote settlement times, maybe we can make this dynamic, but didn't want to do it without the details on that, but open to other ideas.

Reply:
I implemented another approach which uses placeholders for upper and lower bounds. So the two approaches are:
1. Using enum of available settlement time strings with hardcoded ranges
2. A more dynamic structure that supports any upper / lower bounds

1 is sufficient for now and also if we just use a predefined set of time ranges, but 2 sets us up better for when we do times based on what the quote says. Although a couple of things with 2:
- requires separate strings to handle plurals (1 hr vs 24 hrs) and different units, unless we switch to use something like [ICU message format](https://unicode-org.github.io/icu/userguide/format_parse/messages/); 
- makes an assumption that ranges always use the same units

Reply:
I think one question that needs to be answered is what level of granularity is actually useful. I think I would lean towards option 1, with a sufficiently broad range of enum values that covers most of our bases. For example, we could have a couple of enum values for hour-based ranges, a couple for day-based ranges, etc. etc. We would then pick which one to use dynamically based off the quote's reported settlement time.. I think the fully dynamic approach with template strings, while ideally it conveys the maximum amount of information, would be a fair amount of work + lots of headache to implement correctly with translations, and be of questionable marginal value over option 1...

Reply:
I also lean towards option 1 for the degree of granularity. More specific would seem odd to me as a user

Reply:
"Your transaction request will be completed between 34 minutes and 5 days" 😖 🙃